src/scenes/App/components/TermsAndConditions/index.jsx

```diff
@@ -0,0 +1,274 @@
+      </ol>
+    </div>
+  );
+
+  renderRepresentations = () => (
+    <div>
+      <Typography variant="title">
+        <FormattedMessage id="tnc.representations" defaultMessage="Your Representations" />
+      </Typography>
+      <ol>
+        <li><FormattedMessage id="tnc.representations1" defaultMessage="You represent, warrant, acknowledge and undertake that: (i) You have verified and determined that Your use of the Site and/or Platform does not violate any laws or regulations of any jurisdiction that applies to You, (ii) You are solely responsible for recording, paying and accounting to any relevant governmental, taxation or other authority for any tax or other levy that may be payable due to Your use of the Site and/or Platform, (iii) You will use the Site and Platform in good faith towards Bodhi and others using the Site and/or Platform, (iv) Bodhi may, at its sole and absolute discretion, decide whether to allow You to use the Site and/or Platform and/or whether to cease Your use of the Site and/or Platform, (v) BOT is offered as a utility token only and is not offered for speculative purposes, and (vi) You shall indemnify Us and hold Us harmless, from and against all claims, liabilities, damages, losses, costs and expenses, including legal fees, arising out of or in connection with any breach of these Terms and Conditions by You, and any other liabilities arising out of Your use of the Site and/or Platform." /></li>
+      </ol>
+    </div>
+  );
+
+  renderAwards = () => (
+    <div>
+      <Typography variant="title">
+        <FormattedMessage id="tnc.awards" defaultMessage="Awards Policy" />
+      </Typography>
+      <ol>
+        <li><FormattedMessage id="tnc.awards1" defaultMessage="All promotions, prizes, awards or special offers are subject to promotion-specific terms and conditions. Please review such promotion-specific terms and conditions that are applicable to the relevant promotion." /></li>
+        <li><FormattedMessage id="tnc.awards2" defaultMessage="Bodhi reserves the right to withdraw any promotion, award, prize or special offer at any time. In the event that Bodhi believes You are abusing or attempting to abuse any award or other promotion,  or are likely to benefit through abuse or lack of good faith, then Bodhi may, at its sole and absolute discretion, deny, withhold or withdraw from You any award or promotion and/or rescind any policy with respect to You, either temporarily or permanently." /></li>
+        <li><FormattedMessage id="tnc.awards3" defaultMessage="Bodhi reserves the right to prevent players from certain jurisdictions from participating and being eligible for any or all promotions and awards, at its sole and absolute discretion." /></li>
+      </ol>
+    </div>
+  );
+
+  renderPowers = () => (
```

Comment:
This should be a component, not a method. Make a stateless component please.

Reply:
done